streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalProcessorContextImpl.java

```diff
@@ -52,7 +52,7 @@ public GlobalProcessorContextImpl(final StreamsConfig config,
     @SuppressWarnings("unchecked")
     @Override
     public StateStore getStateStore(final String name) {
-        final StateStore store = stateManager.getGlobalStore(name);
+        final StateStore store = stateManager.getStore(name);
```

Comment:
I'm going to collapse the GlobalStateManagerImpl into ProcessorStateManager as well, so this is just a cleanup -- would file another PR for this part soon.